### ISSUE TITLE
More than two conditions in WHERE results in an error

### ISSUE THREAD

**Issue by dropsonic** [AmrDeveloper/GQL#75]
**Describe the bug**
More than two conditions in WHERE results in an error.

**To Reproduce**
Execute any statement with more than two conditions in the WHERE section:

```sql
SELECT COUNT(commit_id) FROM commits WHERE datetime >= "2023-01-01 00:00:00" AND email = "myname@mail.com" AND comment like "%merge%"
```

**Expected behavior**
The results are returned; all condition are applied.

**Actual behavior**
An error: `Unexpected content after the end of 'SELECT' statement`

**Screenshots**
![image](https://github.com/AmrDeveloper/GQL/assets/1544021/742fbbde-5cb3-4ede-ae62-63f13f3373ae)

**GQL (please complete the following information):**
 - Version v0.12.0

**Comment by AmrDeveloper** [comment]
Hello @dropsonic,

Thank you for reporting, i solved it now for AND, OR, XOR

Will do more testing and release it in 0.13.0 next friday

Thanks

**Comment by AmrDeveloper** [comment]
Fixed and released in `0.13.0`

https://github.com/AmrDeveloper/GQL/releases/tag/0.13.0

Thank you
